app/views/registrations/_form.haml

```diff
@@ -29,6 +29,8 @@
 
     - if AppConfig.settings.captcha.enable?
       = show_simple_captcha :object => 'user', :code_type => 'numeric'
+      :javascript
+        $('#user_captcha').addClass('form-control');
```

Comment:
Yeah, I remember having been distrubed with this case. Unhopfully, [the man page](https://github.com/kikyous/simple-captcha2) don't give any clue about adding classes. I'll try your proposition, @jhass